perception/traffic_light_recognition/traffic_light_ssd_fine_detector/include/traffic_light_ssd_fine_detector/nodelet.hpp

```diff
@@ -53,45 +50,45 @@ typedef struct Detection
 
 namespace traffic_light
 {
-class TrafficLightSSDFineDetectorNodelet : public nodelet::Nodelet
+class TrafficLightSSDFineDetectorNodelet : public rclcpp::Node
```

Comment:
If I were to change the class name, I would also want to change the file name.
However, I don't want to have changes in the file name to avoid conflicts when we merge updated ROS1 codes so let's keep it as is for now.

Reply:
Sounds good for me.